changelog.md

```diff
@@ -1,3 +1,11 @@
+### v4.1.4
```

Comment:
Maybe it is worth updating version in `cargo.toml` too, so it will not require separate push